nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ControlRate.java

```diff
@@ -381,6 +392,14 @@ public boolean tryAdd(final long value) {
 
     private class ThrottleFilter implements FlowFileFilter {
 
+        private final long flowFilesPerTrigger;
+        private final AtomicLong flowFilesFiltered = new AtomicLong(0L);
```

Comment:
Yep, I picked up the pattern from Throttle, but this doesn't serve the same purpose.